### PR TITLE
[4.1] com_media: hardcoded language strings

### PR DESCRIPTION
**Pull request by pe7er**
The admin component **com_media** had some hard coded English texts.

### Summary of Changes
This PR changes the hard coded English texts to language strings 
and adds the translations to administrator/language/en-GB/com_media.ini

### CODE REVIEW

**Comment by brianteeman** [comment]
I may be wrong but my undeerstanding was that exception messages were not being translated

**Comment by pe7er** [comment]
I found the hard coded text when I was looking for the "Media Provider not found" that a J4 user reported to me when they tried to use Media as Custom Field Type. 

If a decision was made to not translate exception messages, then that could explain why these strings weren't translated yet :-)

**Comment by brianteeman** [comment]
It appears to be inconsistent even within the same file and peersonally I don't understand why everything isnt a translatable string. 

For example cheeck out https://github.com/joomla/joomla-cms/blob/4.1-dev/plugins/filesystem/local/src/Adapter/LocalAdapter.php

**Comment by laoneo** [comment]
A basic rule should be that the API throws a specific exception and the representation layer should catch it and display a meaningful error message. We tried to do this in the local adapter with the `FileNotFoundException` for example. But looks like some slipped through our fingers.

**Comment by brianteeman** [comment]
some? 12/15 is not some

File: administrator/language/en-GB/com_media.ini [hunk 1]
```diff
@@ -24,6 +24,11 @@ COM_MEDIA_DELETE_SUCCESS="Item deleted."
 COM_MEDIA_DROP_FILE="Drop file(s) to Upload"
 COM_MEDIA_EDIT="Media Edit"
 COM_MEDIA_ERROR="An error occurred."
+COM_MEDIA_ERROR_ACCOUNT_NOT_FOUND="The account was not found"
```
Comment (brianteeman) [inline administrator/language/en-GB/com_media.ini:27]:
```suggestion
COM_MEDIA_ERROR_ACCOUNT_NOT_FOUND="The account was not found."
```

File: administrator/language/en-GB/com_media.ini [hunk 2]
```diff
@@ -24,6 +24,11 @@ COM_MEDIA_DELETE_SUCCESS="Item deleted."
...
+COM_MEDIA_ERROR_ACCOUNT_NOT_SET="Account was not set"
```
Comment (brianteeman) [inline administrator/language/en-GB/com_media.ini]:
```suggestion
COM_MEDIA_ERROR_ACCOUNT_NOT_SET="Account was not set."
```

File: administrator/language/en-GB/com_media.ini [hunk 3]
```diff
@@ -24,6 +24,11 @@ COM_MEDIA_DELETE_SUCCESS="Item deleted."
...
+COM_MEDIA_ERROR_MEDIA_PROVIDER_NOT_FOUND="Media Provider not found"
```
Comment (brianteeman) [inline administrator/language/en-GB/com_media.ini]:
```suggestion
COM_MEDIA_ERROR_MEDIA_PROVIDER_NOT_FOUND="Media Provider not found."
```

File: administrator/language/en-GB/com_media.ini [hunk 4]
```diff
@@ -24,6 +24,11 @@ COM_MEDIA_DELETE_SUCCESS="Item deleted."
...
+COM_MEDIA_ERROR_NO_ADAPTER_FOUND="No adapter found"
```
Comment (brianteeman) [inline administrator/language/en-GB/com_media.ini]:
```suggestion
COM_MEDIA_ERROR_NO_ADAPTER_FOUND="No adapter found."
```

**Comment by pe7er** [comment]
Thanks @brianteeman!

**Comment by brianteeman** [comment]
I have tested this item :white_check_mark: successfully on 9a915cf4b846a1ea95ffd0ef9aa2f5289028a76a<br /><br />Marking as a successful test as it does what it says. Maintainers can decide if we do want to translate exceptions and then hopefully come up with a plan to update all existing hard coded exception strings.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37588">issues.joomla.org/tracker/joomla-cms/37588</a>.</sub>

**Comment by pe7er** [comment]
Thanks. If this PR gets accepted, then I'll fix more of those hard coded Exception errors in components when I find them.

**Comment by Shubhamverma2796** [comment]
I have tested this item :white_check_mark: successfully on 9a915cf4b846a1ea95ffd0ef9aa2f5289028a76a<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37588">issues.joomla.org/tracker/joomla-cms/37588</a>.</sub>

**Comment by richard67** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37588">issues.joomla.org/tracker/joomla-cms/37588</a>.</sub>

**Pull request merged**